R/logging.r

```diff
@@ -10,26 +10,29 @@
 #'
 #' @name log_*
 #' @param ...  message to log
+#' @importFrom crayon green
 #' @export
 log_debug <- function(...) {
     if (isTRUE(getOption('jupyter.log_level') >= 3L)) {
-        log_msg('DEBUG', sprintf(...))
+        log_msg(green('DEBUG'), sprintf(...))
```

Comment:
IMO the coloring should be done in log_msg_stderror as as far as I can set will now output escape chars into the text based logfile.